topics/modules-api-ref.md

```diff
@@ -1732,22 +2152,40 @@ Example code fragment:
        // Do something with myval.
      }
 
-This API is documented here: [https://redis.io/topics/modules-intro](https://redis.io/topics/modules-intro)
+This API is documented here:[ https://redis.io/topics/modules-intro]( https://redis.io/topics/modules-intro)
```

Comment:
These links look wrong. It's caused by a change in the ruby script here: https://github.com/redis/redis/pull/8442/files#r615932389

Reply:
The links work anyway, so no disaster. The generated HTML code for this link is `<p>This API is documented here:<a href="https://redis.io/topics/modules-intro"> https://redis.io/topics/modules-intro</a></p>`.

Reply:
Fixed here: https://github.com/redis/redis/pull/8818